test/runtests.jl

```diff
@@ -95,6 +105,21 @@ using AtomsCalculators.AtomsCalculatorsTesting
         return ( forces = f, )
     end
 
+    AtomsCalculators.@generate_interface function AtomsCalculators.calculate(
+        ::AtomsCalculators.Hessian, 
+        system, 
+        calculator::MyTypeC; 
+        kwargs...
+    )
+        # we can ignore kwargs... or use them to tune the calculation
+        # or give extra information like pairlist
+    
```

Comment:
Is this not boilerplate ?

Reply:
Yes, it is. Originally the idea was to add that there, so that, if someone looks for implementation examples, there would be some "helpful comments".